orb_slam_2/src/ORBextractor.cc

```diff
@@ -73,1062 +72,1129 @@ const int PATCH_SIZE = 31;
-    -12,1, -6,1/*mean (0.243297), correlation (0.489447)*/,
-    3,2, 4,-8/*mean (0.00155196), correlation (0.553496)*/,
-    -10,-10, -10,9/*mean (0.00239541), correlation (0.54297)*/,
-    8,-13, 12,12/*mean (0.0034413), correlation (0.544361)*/,
-    -8,-12, -6,-5/*mean (0.003565), correlation (0.551225)*/,
-    2,2, 3,7/*mean (0.00835583), correlation (0.55285)*/,
-    10,6, 11,-8/*mean (0.00885065), correlation (0.540913)*/,
-    6,8, 8,-12/*mean (0.0101552), correlation (0.551085)*/,
-    -7,10, -6,5/*mean (0.0102227), correlation (0.533635)*/,
-    -3,-9, -3,9/*mean (0.0110211), correlation (0.543121)*/,
-    -1,-13, -1,5/*mean (0.0113473), correlation (0.550173)*/,
-    -3,-7, -3,4/*mean (0.0140913), correlation (0.554774)*/,
-    -8,-2, -8,3/*mean (0.017049), correlation (0.55461)*/,
-    4,2, 12,12/*mean (0.01778), correlation (0.546921)*/,
-    2,-5, 3,11/*mean (0.0224022), correlation (0.549667)*/,
-    6,-9, 11,-13/*mean (0.029161), correlation (0.546295)*/,
-    3,-1, 7,12/*mean (0.0303081), correlation (0.548599)*/,
-    11,-1, 12,4/*mean (0.0355151), correlation (0.523943)*/,
-    -3,0, -3,6/*mean (0.0417904), correlation (0.543395)*/,
-    4,-11, 4,12/*mean (0.0487292), correlation (0.542818)*/,
-    2,-4, 2,1/*mean (0.0575124), correlation (0.554888)*/,
-    -10,-6, -8,1/*mean (0.0594242), correlation (0.544026)*/,
-    -13,7, -11,1/*mean (0.0597391), correlation (0.550524)*/,
-    -13,12, -11,-13/*mean (0.0608974), correlation (0.55383)*/,
-    6,0, 11,-13/*mean (0.065126), correlation (0.552006)*/,
-    0,-1, 1,4/*mean (0.074224), correlation (0.546372)*/,
-    -13,3, -9,-2/*mean (0.0808592), correlation (0.554875)*/,
-    -9,8, -6,-3/*mean (0.0883378), correlation (0.551178)*/,
-    -13,-6, -8,-2/*mean (0.0901035), correlation (0.548446)*/,
-    5,-9, 8,10/*mean (0.0949843), correlation (0.554694)*/,
-    2,7, 3,-9/*mean (0.0994152), correlation (0.550979)*/,
-    -1,-6, -1,-1/*mean (0.10045), correlation (0.552714)*/,
-    9,5, 11,-2/*mean (0.100686), correlation (0.552594)*/,
-    11,-3, 12,-8/*mean (0.101091), correlation (0.532394)*/,
-    3,0, 3,5/*mean (0.101147), correlation (0.525576)*/,
-    -1,4, 0,10/*mean (0.105263), correlation (0.531498)*/,
-    3,-6, 4,5/*mean (0.110785), correlation (0.540491)*/,
-    -13,0, -10,5/*mean (0.112798), correlation (0.536582)*/,
-    5,8, 12,11/*mean (0.114181), correlation (0.555793)*/,
-    8,9, 9,-6/*mean (0.117431), correlation (0.553763)*/,
-    7,-4, 8,-12/*mean (0.118522), correlation (0.553452)*/,
-    -10,4, -10,9/*mean (0.12094), correlation (0.554785)*/,
-    7,3, 12,4/*mean (0.122582), correlation (0.555825)*/,
-    9,-7, 10,-2/*mean (0.124978), correlation (0.549846)*/,
-    7,0, 12,-2/*mean (0.127002), correlation (0.537452)*/,
-    -1,-6, 0,-11/*mean (0.127148), correlation (0.547401)*/
+static int bit_pattern_31_[256 * 4] = {
+    8, -3, 9, 5/*mean (0), correlation (0)*/, 4, 2, 7,
+    -12/*mean (1.12461e-05), correlation (0.0437584)*/, -11, 9, -8,
```

Comment:
another reason not to do auto-formatting - please revert the effects that auto-formatting has on this code block. Note that if your actual changes are few enough, it might be more efficient to just revert all changes in this PR, then re-add your changes without auto-formatting